app-arch/bzip2/bzip2-1.0.6-r10.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic multilib-minimal toolchain-funcs
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
+HOMEPAGE="http://www.bzip.org/"
+SUSE_PV="${PV}.0.2"
+SUSE_P="${PN}-${SUSE_PV}"
+SRC_URI="http://ftp.suse.com/pub/people/sbrabec/bzip2/tarballs/${SUSE_P}.tar.gz"
+S="${WORKDIR}/${SUSE_P}"
+
+LICENSE="BZIP2"
+SLOT="0/1" # subslot = SONAME
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd"
+IUSE="static static-libs"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.0.6-progress.patch
+	"${FILESDIR}"/${PN}-1.0.4-POSIX-shell.patch #193365
+	"${FILESDIR}"/${PN}-1.0.6-mingw.patch #393573
+	"${FILESDIR}"/${PN}-1.0.6-CVE-2016-3189.patch #620466
+)
+
+DOCS=( manual.pdf CHANGES NEWS README.autotools README README.XML.STUFF )
+HTML_DOCS=( manual.html )
+
+multilib_src_configure() {
+	# use static && append-ldflags -static # doesn't seem to work...
+	# fails at gen_usr_ldscript because adding -static to ldflags
+	# prevents generation of libbz2.so
+
+	ECONF_SOURCE="${S}" econf \
+		$(use_enable static-libs static)
+}
+
+multilib_src_install() {
```

Comment:
Please bring back https://github.com/gentoo/gentoo/blob/master/app-arch/bzip2/bzip2-1.0.6-r9.ebuild#L65-L75 aka https://bugs.gentoo.org/338321

Reply:
Gotcha. I'll see what I can do in the current git to make this not needed in the ebuild.

Reply:
Simple testing shows those symlinks do get properly built with my git repo
for bzip2, so a 'release' from it should work fine. Current ebuild (pushing)
gives these results:
```diff
--- /dev/fd/63	2018-06-23 00:10:53.209351862 -0500
+++ /dev/fd/62	2018-06-23 00:10:53.209351862 -0500
@@ -1,4 +1,4 @@
-/tmp/portage/app-arch/bzip2-1.0.6-r9/image/
+/tmp/portage/app-arch/bzip2-1.0.6-r10/image/
 ├── bin
 │   ├── bunzip2 -> bzip2
 │   ├── bzcat -> bzip2
@@ -24,32 +24,35 @@
     │   ├── libbz2.so -> libbz2.so.1.0.6
     │   ├── libbz2.so.1 -> libbz2.so.1.0.6
     │   ├── libbz2.so.1.0 -> libbz2.so.1.0.6
-    │   └── libbz2.so.1.0.6
+    │   ├── libbz2.so.1.0.6
+    │   └── pkgconfig
+    │       └── bzip2.pc
     ├── lib64
     │   ├── libbz2.a
-    │   └── libbz2.so
+    │   ├── libbz2.so
+    │   └── pkgconfig
+    │       └── bzip2.pc
     └── share
         ├── doc
-        │   └── bzip2-1.0.6-r9
-        │       ├── CHANGES.bz2
+        │   └── bzip2-1.0.6-r10
         │       ├── html
         │       │   └── manual.html
         │       ├── manual.pdf
+        │       ├── NEWS.bz2
+        │       ├── README.autotools.bz2
         │       ├── README.bz2
-        │       ├── README.COMPILATION.PROBLEMS.bz2
         │       └── README.XML.STUFF.bz2
         └── man
             └── man1
-                ├── bunzip2.1.bz2 -> bzip2.1.bz2
-                ├── bzcat.1.bz2 -> bzip2.1.bz2
-                ├── bzcmp.1.bz2 -> bzdiff.1.bz2
+                ├── bunzip2.1
+                ├── bzcat.1
+                ├── bzcmp.1
                 ├── bzdiff.1.bz2
-                ├── bzegrep.1.bz2 -> bzgrep.1.bz2
-                ├── bzfgrep.1.bz2 -> bzgrep.1.bz2
+                ├── bzegrep.1
+                ├── bzfgrep.1
                 ├── bzgrep.1.bz2
                 ├── bzip2.1.bz2
-                ├── bzip2recover.1.bz2 -> bzip2.1.bz2
-                ├── bzless.1.bz2 -> bzmore.1.bz2
+                ├── bzless.1
                 └── bzmore.1.bz2
 
-13 directories, 39 files
+15 directories, 40 files
```